libopenage/renderer/renderer.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015-2019 the openage authors. See copying.md for legal info.
+
+#include "renderer.h"
+
+namespace openage {
+namespace renderer {
+
+RenderPass::RenderPass(std::vector<Renderable> renderables, RenderTarget const* target)
+:renderables(renderables), target(target) {}
+
+std::vector<Renderable> RenderPass::get_renderables() const {
```

Comment:
I don't think an OO-interface for `RenderPass` is necessary, seeing as all members are exposed with get/set anyway. The plain struct from earlier was sufficient and less code to maintain.

Reply:
The main reason I introduced an OO-interface is the `is_optimized` field, cause it wouldn't make sense to explicitly set `is_optimized` to `false` each time you create a renderpass